src/classes/Settler.java

```diff
@@ -0,0 +1,34 @@
+package classes;
+
+import java.util.Collection;
+
+public class Settler extends Spaceship implements Steppable{
+    Collection<Bills> bills = new Collection<Bills>();
+    Collection<Material> carriedMaterial = new Collection<Material>();
+    private TeleportPair teleportpair;
+
+    public Material GetCarriedMaterial(){
+        return carriedMaterial.get(0);
+    }
+    public void GetHitByExplosion(){};
+    public void Drill(){};
+    public void Mine(){};
+    public String GetType(){
```

Comment:
Drill ele "@Override" es minden fgv ele ami az ososztalyban is van, csak maskepp valositja meg (Ha szukseges)